.github/workflows/ci-main.yaml

```diff
@@ -227,7 +227,113 @@ jobs:
             --sc4s_host=sc4s \
             --junitxml=test-results/test.xml \
             -n 14 \
-            -k 'not lite'
+            -k 'not lite and not name_cache'
+  
+  test-ipv4-name-cache:
```

Comment:
Probably we need to add it as [dependency for release](https://github.com/splunk/splunk-connect-for-syslog/blob/a73020bf0bbae984cf3aa229267d9366a102f723/.github/workflows/ci-main.yaml#L362), because when it fail release anyway will be done